src/screens/Learn/Faq/Faq.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { Wrapper, PageHeader } from './Faq.style';
+import FaqSection from './FaqSection';
+import { faqContent } from 'cms-content/learn';
+
+const faqHeader = faqContent.header;
+const faqSections = faqContent.sections;
+
+const Faq = () => {
+  return (
+    <Wrapper>
+      <PageHeader>{faqHeader}</PageHeader>
+      <p>*** insert table of contents ***</p>
+      {faqSections.map((section: any, i: number) => (
+        <FaqSection content={section} />
```

Comment:
Consider fixing the type for the `section` and adding keys for each section (you can use the section ID, that's better than using the index)